share/doc/homebrew/External-Commands.md

```diff
@@ -87,11 +87,11 @@ These commands have been contributed by Homebrew users but are not included in t
 >Scrapes a formula's homepage to get more information: [https://gist.github.com/475200](https://gist.github.com/475200)
 
 ### brew-services
->Simple support to start formulae using launchctl, has out of the box support for any formula which defines `startup_plist` (e.g. mysql, postgres, redis u.v.m.): [https://github.com/gapple/homebrew-services](https://github.com/gapple/homebrew-services)
+>Simple support to start formulae using launchctl, has out of the box support for any formula which defines `startup_plist` (e.g. mysql, postgres, redis u.v.m.): [https://github.com/Homebrew/homebrew-services](https://github.com/Homebrew/homebrew-services)
 
 > Install using:
 > ```
-  $ brew tap gapple/services
+  $ brew tap homebrew/services
```

Comment:
This isn't actually even needed any more; people can just run `brew cask`, `brew services` and `brew bundle` and they'll auto install.